packages/core/useSwipe/index.ts

```diff
@@ -15,21 +16,22 @@ export enum SwipeDirection {
  * @param options
  */
 export function useSwipe(
-  target: MaybeRef<HTMLElement | null | undefined>,
-  options?: {
+  target: MaybeRef<EventTarget>,
+  options: {
     threshold?: number
+    preventScrolling: boolean
     onSwipe?: (e: TouchEvent) => void
     onSwipeEnd?: (e: TouchEvent, direction: SwipeDirection) => void
-  },
+  } = { preventScrolling: false },
 ) {
-  const targetRef = ref(target)
   const { threshold, onSwipe, onSwipeEnd } = options || {}
```

Comment:
```suggestion
  const { threshold, onSwipe, onSwipeEnd, preventScrolling = true } = options || {}
```

Reply:
And why not being passive by default?

Reply:
passive by default would be `preventScrolling` = `false` ;)

Reply:
how about just name this option to `passive` and default to true?